scarb/src/core/manifest/mod.rs

```diff
@@ -49,6 +49,9 @@ pub struct Manifest {
     pub compiler_config: ManifestCompilerConfig,
     #[builder(default)]
     pub scripts: BTreeMap<SmolStr, ScriptDefinition>,
+    /// Allow experimental features.
+    #[builder(default)]
+    pub experiments: Option<HashSet<String>>,
```

Comment:
```suggestion
    pub experiments: Option<HashSet<SmolStr>>,
```

My intuition says that `SmolStr` would be a better choice here, because it avoids allocations and has cheap clones.